test/unit/transloadit/test_api.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+describe Transloadit::ApiModel do
+  before do
+    @transloadit = Transloadit.new(:key => '')
+  end
+
+  it 'must allow initialization' do
+    Transloadit::ApiModel.new(@transloadit).
+      must_be_kind_of Transloadit::ApiModel
+
+    Transloadit::Template.new(@transloadit).
+      must_be_kind_of Transloadit::Template
+  end
+
+  describe 'when initialized' do
+    before do
+      @foo = 'foo'
+      @bar = 'bar'
+
+      @api = Transloadit::ApiModel.new @transloadit,
+        :foo => @foo,
+        :bar => @bar
+    end
```

Comment:
minitest also supports `let`, which I find easier to grasp when reading tests. 

```ruby
let(:foo) { 'foo' }
let(:bar) { 'bar' }
let(:api) { Transloadit::ApiModel.new(
  @transloadit,
  :foo => foo,
  :bar => bar
)}

it 'must store a pointer to the transloadit instance' do
  api.transloadit.must_equal @transloadit
end
```

that way, one can easily see where the variables have been declared. What do you think about that?